src/components/OutboundLink.js

```diff
@@ -0,0 +1,31 @@
+var React = require('react');
+var assign = require('react/lib/Object.assign');
+
+var OutboundLink = React.createClass({
+  displayName: 'OutboundLink',
+  statics: {
+    outboundTrackingFunction: function() {
```

Comment:
thinking about it, it's probably worth renaming this to something shorter like `trackLink`